lib/improver/percentile.py

```diff
@@ -59,7 +57,7 @@ def __init__(self, collapse_coord, percentiles=None):
         collapse_coord : str (or list of str)
             The name of the coordinate(s) to collapse over.
 
-        percentiles : list (optional)
+        percentiles : tuple or None (optional)
```

Comment:
Does this just have to be a iterable? I'm guessing it can still be a list, but I'm not sure?

Reply:
Done.